{{cookiecutter.project_slug}}/docker-compose.yml

```diff
@@ -9,6 +9,5 @@ services:
     volumes:
       - "./:/app"
     environment:
-      - PYTHONUNBUFFERED=1
       - ENVIRONMENT=${ENVIRONMENT:-development}
-      - SECRET_KEY=${SECRET_KEY:-ohgodpleasechangeme}
+    command: gunicorn -c gunicorn.py {{cookiecutter.project_slug}}.wsgi:app
```

Comment:
I'm not opposed, I just don't get the reasoning why the command is better here than in the `Dockerfile` directly.

Reply:
The command will sometimes differ between environments and in those cases not having a default can perhaps help you discover a misconfiguration faster (doesn't run instead of running with incorrect parameters). That's the reasoning but I don't really feel strongly about it